herald/src/data/generate-guild-data.js

```diff
@@ -0,0 +1,72 @@
+"use strict"
```

Comment:
Suggest to update the `npm build` script in `herald/package.json` to execute this script automatically (before `ng build`), so each person doesn't need to remember to do it themselves.